### PR TITLE
Update maximal70.ged

### PR DESCRIPTION
**Pull request by tychonievich**
Fix the double HEAD.NOTE introduced earlier by mistake

### CODE REVIEW

File: testfiles/gedcom70/maximal70.ged [hunk 1]
```diff
@@ -2,6 +2,15 @@
 1 GEDC
 2 VERS 7.0
 1 NOTE This file is intended to provide coverage of parts of the specification and does not contain meaningful historical or genealogical data.
+2 CONT That observation applies even to the substructures of this NOTE.
```
Comment (tychonievich) [inline testfiles/gedcom70/maximal70.ged:5]:
```suggestion
2 CONT That observation applies even to the substructures of this NOTE; for example, the TRAN does not translate this text, it merely exercises the presence of a TRAN.
```

Comment (dthaler) [reply testfiles/gedcom70/maximal70.ged:5]: Another possible translatable phrase: "See the specification for advice on how to use" / "...advise"

File: testfiles/gedcom70/maximal70.ged [hunk 2]
```diff
@@ -2,6 +2,15 @@
...
+2 MIME text/plain
+2 LANG en-US
+2 TRAN British English
+3 LANG en-GB
```
Comment (tychonievich) [inline testfiles/gedcom70/maximal70.ged:9]:
```suggestion
1 TRAN Diese Datei soll Teile der Spezifikation abdecken und enthält keine aussagekräftigen historischen oder genealogischen Daten.
2 LANG de
```

File: testfiles/gedcom70/maximal70.ged (hunk 1 above)
Comment (tychonievich) [inline testfiles/gedcom70/maximal70.ged:5]:
```suggestion
```

**Review by dthaler — APPROVED** [review]

**Comment by dthaler** [comment]
Approved during GEDCOM Steering Committee meeting

**Pull request merged**